v21.2/ui-overview.md

```diff
@@ -65,6 +65,17 @@ You can access the DB Console from every node at `http://<host>:<http-port>`, or
 
 For guidance on accessing the DB Console in the context of cluster deployment, see [Start a Local Cluster](start-a-local-cluster.html) and [Manual Deployment](manual-deployment.html).
 
+### Proxy DB Console
+
+If your CockroachDB cluster is behind a load balancer, you may wish to proxy your DB Console connection to a different node in the cluster from the node you first connect to. You can accomplish this using one of these methods:
```

Comment:
```suggestion
If your CockroachDB cluster is behind a load balancer, you may wish to proxy your DB Console connection to a different node in the cluster from the node you first connect to. This is useful in deployments where a third-party load balancer otherwise determines which CockroachDB node you connect to in DB Console, or where web management access is limited to a subset of CockroachDB instances in a cluster.

You can accomplish this using one of these methods:
```

Reply:
Just to note @andf-crl I assume Michael's suggestions would apply to v22.1 as well.